db/migrations/sdp-migrations/2025-05-18.1-create-embedded-wallet.sql

```diff
@@ -0,0 +1,24 @@
+-- +migrate Up
+
+CREATE TYPE embedded_wallet_status AS ENUM(
+    'PENDING',
+    'PROCESSING',
+    'FAILED',
+    'SUCCESS'
+);
+
+CREATE TABLE embedded_wallets (
+    token VARCHAR(36) PRIMARY KEY DEFAULT public.uuid_generate_v4(),
+    tenant_id VARCHAR(36) NOT NULL,
+    wasm_hash VARCHAR(64),
+    contract_address VARCHAR(56),
+    created_at TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT NOW(),
+    updated_at TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT NOW(),
+    wallet_status embedded_wallet_status NOT NULL DEFAULT 'PENDING'::embedded_wallet_status
+);
+
```

Comment:
To get the updated_at to get updated automatically, you'll need to add a trigger, like in:

https://github.com/stellar/stellar-disbursement-platform-backend/blob/3b6bd647f115876b7ecdd5e240ae81253427f272/db/migrations/sdp-migrations/2023-01-30.0-update-disbursements-table.sql#L50-L51